<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryCardSettingsWidget</class>
 <widget class="QWidget" name="MemoryCardSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="slotGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Memory Card Slots</string>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cardGroupBox">
     <property name="title">
      <string>Memory Cards</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="directory"/>
        </item>
        <item>
         <widget class="QPushButton" name="browse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open">
          <property name="text">
           <string>Open...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="MemoryCardListWidget" name="cardList">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Formatted</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Modified</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="refreshCard">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="createCard">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="renameCard">
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="convertCard">
          <property name="text">
           <string>Convert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteCard">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="settingsGroupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="automaticManagement">
        <property name="text">
         <string>Automatically manage saves based on running game</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MemoryCardListWidget</class>
   <extends>QTreeWidget</extends>
   <header>Settings/MemoryCardSettingsWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
